app/config/settings.py

```diff
@@ -121,6 +121,9 @@ def strtobool(val) -> bool:
     "DOCUMENTATION_HELP_FORUM_SLUG", "general"
 )
 
+# Home page news blog tag
+HOMEPAGE_NEWS_BLOG_TAG = os.environ.get("HOMEPAGE_NEWS_BLOG_TAG", "homenews")
+
```

Comment:
Ideally, we would use the 'news' or 'announcement' tag rather than 'homenews', but the former two are already in use on GC, hence the slightly strange tag here. Open for different suggestions as well.

Reply:
How about highlight?

Reply:
You could also use a model attribute for this, see my comment about Algorithms and Challenges too

Reply:
Yes, good idea, that's better, less chance for errors / confusion.